src/ArtifactPaths.jl

```diff
@@ -23,11 +23,15 @@ function get_lookup_filename(optics_type::Symbol, λ::Symbol)
     @assert optics_type ∈ (:gas, :cloud, :aerosol)
     @assert λ ∈ (:lw, :sw)
     basedir = get_artifact_path()
+    currdir = Base.@__DIR__
```

Comment:
```suggestion
    currdir =  @__DIR__
```

Base is not needed, in case you wanted to remove the qualification